internal/loader/configuration_setting_loader.go

```diff
@@ -88,6 +91,7 @@ const (
 	CertTypePfx                           string = "application/x-pkcs12"
 	TlsKey                                string = "tls.key"
 	TlsCrt                                string = "tls.crt"
+	RequestTracingEnabled                 string = "REQUEST_TRACING_ENABLED"
```

Comment:
How does a customer disable the tracing? What about the AKS extension customer?

Reply:
They can set `requestTracing.enabled=false` while installling the proivder. Same to the extension customer.

Reply:
You mean in the yaml file?

Reply:
@zhenlan I mean here https://github.com/Azure/AppConfiguration-KubernetesProvider/pull/47/files#r1652128093

Reply:
This is the parameter a user pass to the helm install? If so, how can an extension user pass the parameter?

Reply:
Extension essentially use helm to do the installation, it supports all the helm values. You can consider there are no difference

Reply:
I understand. My question is how an extension user can customize the helm install command and pass the extra parameter.

Reply:
LIke how replicaCount can be set:
https://learn.microsoft.com/en-us/azure/aks/azure-app-configuration-settings#configure-the-replica-count

User can customize use this command while install the extension.

```

az k8s-extension create --cluster-type managedClusters --cluster-name myAKSCluster --resource-group myResourceGroup --name appconfigurationkubernetesprovider --extension-type Microsoft.AppConfiguration --auto-upgrade-minor-version true --configuration-settings "requestTracing.enabled=false"
```

Reply:
Okay, so it's a different command. I am curious how you plan to explain this to customers.